gradle/quality.gradle

```diff
@@ -13,7 +12,6 @@ tasks.withType(FindBugs) {
 }
 
 checkstyle {
-    toolVersion = "8.14"
```

Comment:
what was this for?

Reply:
We use the default plugin version bundled, saves us from updating it from time-to-time.